plugins/browser-plugin-button-click-tracking/src/types.ts

```diff
@@ -18,6 +18,8 @@ export interface ButtonClickTrackingConfiguration {
   filter?: Filter;
   /** The dynamic context which will be evaluated for each button click event */
   context?: DynamicContext;
+  /** A default label to use if one can not be determined by the content or data attributes */
```

Comment:
If I understand correctly, if this is not specified, the default will be `(empty)`, should we note this here or somewhere else?

Reply:
Maybe this will be more visible in the tracker docs.